robottelo/helpers.py

```diff
@@ -413,3 +414,37 @@ def default_url_on_new_port(oldport, newport):
 def get_func_name(func):
     """Given a func object return standardized name to use across project"""
     return '{0}.{1}'.format(func.__module__, func.__name__)
+
+
+def get_services_status():
+    """Check if core services are running"""
+    major_version = get_host_info()[1]
+    services = (
+        'foreman-proxy',
+        'foreman-tasks',
+        'httpd',
+        'mongod',
+        'postgresql',
+        'pulp_celerybeat',
+        'pulp_resource_manager',
+        'pulp_streamer',
+        'pulp_workers',
+        'qdrouterd',
+        'qpidd',
+        'smart_proxy_dynflow_core',
+        'squid',
+        'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
+    )
```

Comment:
perhaps this service should be gotten from a function like get_host_services or get_sat_services as they are used in more than one place